.travis.yml

```diff
@@ -74,6 +75,11 @@ before_install:
       # TravisCI installs its own cmake to another location which ovverides other installations
       # if they don't call the new binary directly
       alias cmake="/usr/bin/cmake"
+      
+      # Qt 5.14.2
+      PATH=/opt/qt514/bin:${PATH}
```

Comment:
You should add quotes and the brackets for variable isn't required:
`PATH="/opt/qt514/bin:$PATH"`

Reply:
Fixed.